source/macos/AdaptiveCards/AdaptiveCards/Renderers/TextInputRenderer.swift

```diff
@@ -41,14 +41,11 @@ class TextInputRenderer: NSObject, BaseCardElementRendererProtocol {
         }
         
         if inputBlock.getIsMultiline() {
-            let multilineView = ACRMultilineInputTextView(config: config)
+            let multilineView = ACRMultilineInputTextView(config: config, inputElement: inputBlock)
```

Comment:
Why in the convenience init is this inputElement optional?

Reply:
To make initialization of class without supplying InputElement as well. Like in test files, where we don't have an input element

Reply:
Ok, correct me if I'm wrong but we do have fakes right?
If it's only for tests then it's fine to use them there right @blue-stone99

Reply:
Yes good point. My bad that I missed it.
`InputElement` must be treated as required to have InputViews to be created. And right, FakeInputElement must be passed.
Also, make sure there is no  2 different logics for Tests and the real code.